### PR TITLE
feat: deprecate atomics

### PR DESCRIPTION
**Pull request by djejaquino**
## Description

Deprecate Atomic Banks and Atomic Cards

## Testing required outside of automated testing?

- [x] Not Applicable

### Screenshots (if appropriate):

- [x] Not Applicable

## Rollback / Rollforward Procedure

- [x] Roll Forward
- [ ] Roll Back

## Reviewer Checklist

- [ ] Description of Change
- [ ] Description of outside testing if applicable.
- [ ] Description of Roll Forward / Backward Procedure
- [ ] Documentation updated for Change

### CODE REVIEW

File: examples/credit_card.html [hunk 1]
```diff
@@ -55,48 +55,52 @@
...
-        // invoke async BasisTheory Payments API store credit card method
-        BasisTheory.atomicCards.create(paymentInfo).then(function (data) {
-          // and when that is done
-          form.reset(); // we can reset our form
-          handleNewCard(data); // and attach the stored credit card info to the view
-        });
+        // invoke async BasisTheory Create Tokens API
+        BasisTheory.tokens
+          .create({
+            type: 'token',
```
Comment (jleon15) [inline examples/credit_card.html:75]: why type token?

File: e2e/cypress/fixtures/credit_card_api_error.html [hunk 2]
```diff
@@ -21,17 +21,18 @@
...
+          expiration_month,
+          expiration_year,
+          cvc,
         };
 
-        BasisTheory.atomicCards
-          .create(paymentInfo)
+        BasisTheory.tokens
+          .create({
+            type: 'token',
```
Comment (jleon15) [inline e2e/cypress/fixtures/credit_card_api_error.html:33]: why type token?

File: examples/credit_card.html (hunk 1 above)
Comment (djejaquino) [reply examples/credit_card.html:75]: Good 👀

File: e2e/cypress/fixtures/credit_card_api_error.html (hunk 2 above)
Comment (djejaquino) [reply e2e/cypress/fixtures/credit_card_api_error.html:33]: Good 👀

**Review by jleon15 — APPROVED** [review]

**Review by lcschy — APPROVED** [review]

**Pull request merged**